go/client/cmd_simplefs.go

```diff
@@ -70,6 +73,13 @@ func makeSimpleFSPath(g *libkb.GlobalContext, path string) keybase1.Path {
 
 	path = filepath.ToSlash(filepath.Clean(path))
 
+	// Certain users seem to want to use SimpleFS on their
+	// mounted KBFS. This is for those who want to do so
+	// from "/keybase/..."
+	if strings.HasPrefix(path, mountDir) {
+		return keybase1.NewPathWithKbfs(path[len(mountDir):])
```

Comment:
Also, we need the result of the test to distinguish what kind of path to make - otherwise I agree.